plugin/src/sbt-test/paradox/generated-source/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.0
```

Comment:
Wouldn't it be better to just delete this file? I don't think it's needed, or wanted.

Reply:
I have no preference

Reply:
I think it should be deleted, otherwise we'll have to manually update it every time we upgrade sbt. No other scripted tests have it.

Reply:
actually there are 2 more - updated